README

```diff
@@ -192,5 +192,11 @@ options.
         If defined, OFI will not abort if fabric provider doesn't support every data
         type x op combination, instead it will print a warning.
 
+  Debugging Environment variables:
+
     SMA_DEBUG (default: off)
         If defined enables debugging messages from OpenSHMEM runtime. 
+
+    SMA_TRAP_ON_ABORT (default: off)
```

Comment:
what happened to the part where you process the environment variable in init? Right now it looks like if its supported by your environment it will be used